program/src/error.rs

```diff
@@ -1,80 +1,45 @@
-use num_derive::FromPrimitive;
-use solana_program::{
-    decode_error::DecodeError,
-    msg,
-    program_error::{PrintProgramError, ProgramError},
-};
-use thiserror::Error;
+use pinocchio::program_error::ProgramError;
 
-#[derive(Error, Clone, Debug, Eq, PartialEq, FromPrimitive)]
```

Comment:
Now I understand why the error struct disappear from the client generation.

Reply:
Yep, annoying, but maybe something that can be fixed upstream at some point.